spec/swagger_schemas/filters/list_participant_declarations.rb

```diff
@@ -0,0 +1,33 @@
+LIST_PARTICIPANT_DECLARATIONS_FILTER = {
+  v1: {
+    description: "Refine participant declarations to return.",
+    type: :object,
+    properties: {
+      participant_id: {
+        description: "The unique id of the participant",
+        type: :string,
+        example: "7e5bcdbf-c818-4961-8da5-439cab1984e0",
+      },
+      updated_since: {
+        description: "Return only records that have been updated since this date and time (ISO 8601 date format).",
+        type: :string,
+        example: "2021-05-13T11:21:55Z",
+      },
+    },
+  },
+}.tap { |h|
+  h[:v2] = h[:v1]
+  h[:v3] = h[:v1].deep_dup
+  h[:v3][:properties].merge!({
+    cohort: {
+      description: "Return participant declarations associated to the specified cohort or cohorts. This is a comma delimited string of years.",
+      type: :string,
+      example: "2021,2022",
+    },
+    delivery_partner_id: {
```

Comment:
There are no delivery partners in NPQ; I think this should be `training_status`?

Reply:
Addressed in https://github.com/DFE-Digital/npq-registration/pull/1504/commits/34a5363b2a2a65fd03d701cfb8a0ac92852e44ef. I have not been able to find a reference to `training_status`

Reply:
👍 I think I was looking at the ECFParticipantFilter by accident, looks good now.